NOAHMP-interface/CMakeLists.txt

```diff
@@ -13,7 +13,8 @@ list(APPEND _noahmp_cap_files noahmp/drivers/nuopc/lnd_comp_kind.F90
 
 # CCPP interface
 list(APPEND _noahmp_ccpp_files noahmp/drivers/ccpp/noahmpdrv.F90
-                               noahmp/drivers/ccpp/sfc_diff.f
+	                             noahmp/drivers/ccpp/lnd_iau_mod.F90
+	                             noahmp/drivers/ccpp/sfc_diff.f
                                noahmp/drivers/ccpp/machine.F
```

Comment:
It might be a spaces vs tabs thing, but it's still not lined up here.

Reply:
@BrianCurtis-NOAA it looks like you may need to approve this/mark as resolved before merge. The changes should already be in.